test/test_node.py

```diff
@@ -169,6 +169,46 @@ def test_constant(self):
     np.testing.assert_equal(output["Y"].shape, shape)
     np.testing.assert_almost_equal(output["Y"].flatten(), values)
 
+  def test_conv(self):
+    node_def = helper.make_node("Conv", ["X", "weights"],
+                                ["Y"])
+    x_shape = [1, 5, 4]
+    x = self._get_rnd(x_shape)
+    weight_shape = [5, 3, 2]
+    weights = self._get_rnd(weight_shape)
+    output = run_node(node_def, [x, weights], device="CUDA")
```

Comment:
We might need to use `supports_device` in the backend class to prevent buildbot failure.